Resources/config/templating.xml

```diff
@@ -7,7 +7,7 @@
         <service id="liip_imagine.templating.filter_helper" class="Liip\ImagineBundle\Templating\Helper\FilterHelper">
             <tag name="templating.helper" alias="imagine" />
             <argument type="service" id="liip_imagine.cache.manager" />
-            <deprecated>The "%service_id%" service is deprecated since LiipImagineBundle 2.2 and will be removed in 3.0.</deprecated>
+            <!-- deprecated service, deprecation is set in the container extension -->
```

Comment:
As per Symfony deprecation message, couldn't this have been fixed by just adding "package" and "version" attributes to the `<deprecated>` node?

Reply:
That would work for Symfony 5.1+, but fail for older versions. That was also my initial solution and changed to this.

Reply:
That's a bummer, but your solution works. The PR looks fine to me.

Reply:
Thank you!